src/BaselineOfMatrixBenchmark/BaselineOfMatrixBenchmark.class.st

```diff
@@ -26,13 +26,16 @@ BaselineOfMatrixBenchmark >> baseline: spec [
 				with: [ spec repository: 'github://ObjectProfile/Roassal3/src' ].
 
 			spec 
-				package: 'Matrix-Benchmarks' with: [ spec requires: #('AILinearAlgebra' 'PolyMath' 'CalipeLSBenchmark' 'Roassal3') ];
-				package: 'Matrix-Benchmarks-Tests' with: [ spec requires: #('Matrix-Benchmarks') ].
+				package: 'Matrix-Benchmarks' 			with: [ spec requires: #('AILinearAlgebra' 'PolyMath' 'CalipeLSBenchmark') ];
```

Comment:
@hernanmd I think `Matrix-Benchmarks` should be replaced by `Matrix-Benchmarks-Core` throughout the baseline.

Reply:
Hi @rakki-18 no, the "Core" is just an internal package tag. The Baseline method spec should work without specifying the "Core" tag.

Reply:
Oh right, my bad.